usage-examples/csharp/driver/Examples/Aggregation/GroupTotal.cs

```diff
@@ -0,0 +1,134 @@
+//	:replace-start: {
+//	  "terms": {
+//	    "_order": "order",
+//      "_aggDb": "aggDb"
+//	  }
+//	}
+namespace Examples.Aggregation;
+
+using MongoDB.Driver;
+using MongoDB.Bson;
+
+public class GroupTotal
+{
+    private IMongoDatabase _aggDB;
+    private IMongoCollection<Order> _orders;
+    
+    public void SeedData()
+    {
+        DotNetEnv.Env.TraversePath().Load();
```

Comment:
I'm not a fan of using third-party tools, like DotNetEnv. I mean, it's great, but it's also one more dependency to keep tabs on. IIRC, in the existing test solution, we just read the file?